### ISSUE TITLE
`<pre>` tags contains tabs after

### ISSUE THREAD

**Issue by evrpress** [maizzle/framework#1156]
- Maizzle Version: 4.7.2
- Node.js Version: 21.5.0

Adding a `<pre>` tag with content in a component causes unexpected tabs in the closing tag.

**component.html**
```html
<pre>
	Content
</pre>
```

**output**
```html
		<pre>
	Content
</pre
		>
```

I think most clients render that just fine but we know how email clients may react.

**Comment by cossssmin** [comment]
This is with `prettify: true`, right?

**Comment by evrpress** [comment]
Yes, actually my settings is
```
prettify: {
    useTabs: true,
}
```
but the result is the same

**Comment by cossssmin** [comment]
Yeah, it's a prettier bug unfortunately, here's a [reproduction](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAhAEQPIGEAqAmgAoCiABABYwC2ANgHwA6Uq19zrlcAhgCYNUAem79OqAEYQ+AT07lyqAA4AnOPIXlc0eLHJ0eMiAFcYLBcNXrzivgEsAbgwv2nNi1Y2atOhGajelmoawq4hnjbasH4sQeoWQmGxQlKy4iK0jCwgADQgEEowdtAAzsigPCoqEADuxJUIZSg8dDWGZXkSKjxgANZwMADKPDRwADJ2UHDIAGYtJXCd3X0Dg0o9kwDmyDAqxosgCzR2O3sHcAAeSnAqdqOwLfg3UJV2cE1zdAt5JVt0cABFYwQeCzeYHABWJQugz+gOBoKQn2+IAAjgi4MRqkomiAeCUALRTOB8Em5EC7Hh2OhbbQ0Gg8ZB4uh0cm-KCbf4AQRguzsElMmJuEymYK+B3YdAA6pQ7PASuswHBBo05Y45TImWASh0QA59gBJKCk2CDMC3Qpc42DGAyf5ilGqCALKXdJRMqwLFQOaZ5SZemBYnibBkOg7rFRepmS8mqSYwKV2PgwSjIACsAAYM3k1Oi7GogyHGUjwXkYDwJInk6mkAAmPLGBb4CsfUsgOA0CQk0l8MY8DnGYNwABiEBUDN5WyZEBmMxAAF950A).

Only way around it will be through a regex that corrects this, will add it to the backlog for the other issue with `embeddedLanguageFormatting`, will fix them both in the next release.

**Comment by cossssmin** [comment]
Reported the issue in the `prettier` repo: https://github.com/prettier/prettier/issues/15982

**Comment by evrpress** [comment]
Thanks Cosmin, I think this issue is not critical. I will submit the issue to the correct repo next time. I'm still not sure which tool in Maizzle is responsible for what 😄

**Comment by cossssmin** [comment]
Don’t worry about it, I actually prefer knowing of something not working in Maizzle and then taking it upstream if needed :)

**Comment by evrpress** [comment]
I use this line in [`afterTransformers`](https://maizzle.com/docs/events#aftertransformers) for now:

```js
html = html.replaceAll(/<\/pre([\s\t]+)>/gi, '$1</pre>');
```

This also preserves any indention.

**Comment by cossssmin** [comment]
Yup, landed on something similar, just more generic:

```js
html.replace('/(<.+?[^>])\n(\s+|\t+)?(.+)\n\s+>/g', '$1 $2>')
```

**Comment by cossssmin** [comment]
Yup, doing something similar now:

```js
html.replace(/(<\/pre)\s+>/g, '$1>')
```

**Comment by cossssmin** [comment]
Should be fixed in `v4.7.3` now 👍
